src/System.IO.FileSystem/src/System/IO/FileSystem.AccessControl.Windows.cs

```diff
@@ -0,0 +1,249 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@JeremyKuhne do you agree with the name for this file?

Reply:
I'd make two: `FileSystem.DirectoryCreation.Windows.cs` and `FileSystem.Attributes.Windows.cs`.  That would be much more understandable, imho.

Reply:
Will do